app/jobs/user_import_job.rb

```diff
@@ -3,7 +3,6 @@
 
 class UserImportJob < ApplicationJob
   retry_on Aws::S3::Errors::ServiceError
-  discard_on Import::Users::InvalidSalesforceId
```

Comment:
I'm puzzled why Sidekiq is ignoring this `discard_on` as it definitely works correctly in `SubmisionIngestionJob` where are there several places you want to hard fail ingest because of user error.

Incidentally, if you wanted to raise an error to Rollbar, `discard_on` takes a block, i.e:

```
discard_on Import::Users::InvalidSalesforceId do |job, exception|
  Rollbar.info "Bulk user import failed for: #{job.arguments.first} due to: #{exception.message}"
end
```